UnityProject/Assets/Scripts/UI/Systems/Lobby/Gender.cs

```diff
@@ -1,13 +1,13 @@
 ﻿public enum Gender
 {
+	NonBinary,
```

Comment:
the change in values means players will get their character's gender swaped. You can get away by assigning manually the value so NonBinary = 3, Male = 1, Female = 2.